translations/readme/README.kr.md

```diff
@@ -1,5 +1,12 @@
 # 🟡 GROC &middot; ![GitHub 리포지토리 크기][1] ![언어별 코드 크기][2]
 
+## 다른 언어로 읽기 [여기](translations/readme/).
+
+![🇦🇩 Catalan][3] ·
```

Comment:
Translate the languages list into its own language, but keep this order (as if they were showed in English).